.github/workflows/deploy-site.yaml

```diff
@@ -0,0 +1,57 @@
+---
+name: deploy website
+on:
+  workflow_dispatch: {}
+  workflow_call:
+    secrets:
+      GCP_PROJECT_NAME:
+        required: true
+      BUCKET_NAME:
+        required: true
+      WIP_PROJECT_ID:
+        required: true
+  push:
+    branches:
+      - master
+
+jobs:
+  upload-website:
+    name: Build and deploy website
+    environment: website
```

Comment:
Does this have any effect? I think we can remove the deployment environment here.

Reply:
The environment is required for the WIP authentication

Reply:
I see, that's the reason it's there. I did workload identity federation without an environment before, but the changes required were a bit counterintuitive, and I don't quite remember the details, so let's just leave it as it is.